frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareInstallerCard/SoftwareInstallerCard.tsx

```diff
@@ -163,18 +167,49 @@ const SoftwareActionsDropdown = ({
     }
   };
 
+  let options =
+    installerType === "package"
+      ? [...SOFTWARE_PACKAGE_DROPDOWN_OPTIONS]
+      : [...APP_STORE_APP_DROPDOWN_OPTIONS];
+
+  if (gitOpsModeEnabled) {
+    const tooltipContent = (
+      <>
+        {repoURL && (
+          <>
+            Manage in{" "}
+            <CustomLink
+              newTab
+              text="YAML"
+              variant="tooltip-link"
+              url={repoURL}
+            />
+            <br />
+          </>
+        )}
+        (GitOps mode enabled)
+      </>
+    );
```

Comment:
This is duplicative of the code in GitOpsModeTooltipWrapper but I don't want to do any refactoring of that code while Jacob is out.